src/main/webapp/web.xml

```diff
@@ -6,6 +6,7 @@
     <servlet>
         <servlet-name>index</servlet-name>
         <servlet-class>mate.controller.IndexController</servlet-class>
+        
     </servlet>
     <servlet-mapping>
         <servlet-name>index</servlet-name>
```

Comment:
As you add @WebServlet(urlPatterns = "/") to your IndexController, you can just remove it from web.xml